README.adoc

```diff
@@ -68,3 +68,38 @@ The Keep Go client.
 
 It runs the Keep network’s random beacon and participates in related
 computations.
+
+== Quick installation
```

Comment:
I think this one should be a part of `docs/development/README`

Reply:
Okay, moved to `docs/development/README`